registrations/management/commands/fix_pmtct_registrations.py

```diff
@@ -0,0 +1,43 @@
+from django.core.management.base import BaseCommand
+
+from registrations.models import Registration
+from registrations.tasks import validate_subscribe
+
+
+class Command(BaseCommand):
+    help = ("Find and fixes all PMTCT registrations that aren't validated"
+            "because of the edd date not being populated")
+
+    def handle(self, *args, **kwargs):
+
+        registrations = Registration.objects.filter(
+            validated=False,
+            reg_type__in=("pmtct_prebirth", "pmtct_postbirth")).iterator()
+
+        updates = 0
+
+        for registration in registrations:
+
+            related_regs = Registration.objects.filter(
+                    validated=True,
+                    registrant_id=registration.registrant_id
+                ).exclude(reg_type__in=("pmtct_prebirth", "pmtct_postbirth")).\
+                order_by('-created_at')
+
+            for related_reg in related_regs:
+
+                if related_reg.data.get("edd"):
+                    del registration.data["invalid_fields"]
```

Comment:
Maybe we should have a check here to make sure that edd is the only missing field (and if it's not, maybe just remove edd from the list). Is this a list that's easily parsable, or is it a human readable string? If it's a string I think it will be too finicky and you can just ignore this suggestion.